release/version.txt

```diff
@@ -0,0 +1 @@
+2.4.7
```

Comment:
nit: our next release should be 2.5.0 since we are dropping support for older versions.